src/AblyBroadcaster.php

```diff
@@ -217,6 +219,14 @@ public function getSignedToken($channelName, $token, $clientId, $guardedChannelC
             $iat = $payload['iat'];
             $exp = $payload['exp'];
             $channelClaims = json_decode($payload['x-ably-capability'], true);
```

Comment:
Need to add checks for matching/changed clientId and userId to avoid identity theft